core/impl/src/main/java/com/blazebit/persistence/impl/CriteriaBuilderConfigurationImpl.java

```diff
@@ -1873,6 +1875,12 @@ private void loadEntityManagerIntegrator() {
         }
     }
 
+    private void loadExtensions() {
+        for (CriteriaBuilderConfigurationContributor contributor : ServiceLoader.load(CriteriaBuilderConfigurationContributor.class)) {
```

Comment:
Sort by `ordinal` and then by fully qualified class name to achieve a deterministic ordering.